add stronger disclaimer to README (#254)
astral-sh/ty: README.md

An extremely fast Python type checker and language server, written in Rust.

> [!WARNING]
> ty is pre-release software and not ready for production use. Expect to encounter bugs, missing

> > features, and fatal errors.
> 
> ty is in active development, and we're working hard to make it stable
> and feature-complete.

Reviewer: Do you intend to append this? If not, ignore this.
```suggestion
> features, and fatal errors.
> 
> ty is in active development, and we're working hard to make it stable
and feature-complete.
```

Author: I did this intentionally -- I didn't feel like this last sentence was part of the "warning", more additional commentary? But it could be wrong, if you think it's better to put it all in the warning box, I can easily do that.

Reviewer: No strong preference! 👍

> **This project is still in development and is not ready for production use.**

Reviewer: Should this be removed?

Reviewer 2: Yeah I think I'd replace this and avoid adding a new heading?

Author: I left it in intentionally. I realize it's a bit redundant, but I think it doesn't hurt to be redundant here.

I do think if we use the Warning note to be more prominent, we could probably remove this.

Author: Done

## Status

ty is pre-release software. Expect to encounter bugs, missing features, and fatal errors.

> We are in active development and working hard to make ty stable and feature-complete.

Reviewer 3: ```suggestion
ty is in active development, and we're working hard to make it stable and feature-complete.
```

> ty is in active development, and we're working hard to make it stable and feature-complete.

Reviewer: We _could_ use the "warning" note?

> [!WARNING]
>
> ty is pre-release software. Expect to encounter bugs, missing features, and fatal errors.
> ty is in active development, and we're working hard to make it stable and feature-complete.

Author: Good idea, done
